apps/web/pages/_app.tsx

```diff
@@ -30,23 +30,20 @@ function MyApp(props: AppProps) {
     pageStatus = "500";
   }
   return (
-    <ContractsProvider>
-      <AppProviders {...props}>
-        <DefaultSeo {...seoConfig.defaultNextSeo} />
-        <I18nLanguageHandler />
-        <Head>
-          <script dangerouslySetInnerHTML={{ __html: `window.CalComPageStatus = '${pageStatus}'` }}></script>
-          <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />
-        </Head>
-        {Component.requiresLicense ? (
-          <LicenseRequired>
-            <Component {...pageProps} err={err} />
-          </LicenseRequired>
```

Comment:
Why was this removed?

Reply:
Seems like an artifact of merge conflict resolving, it's not intentional

Reply:
Let's restore this component and we're good to merge 👍🏽